### ISSUE TITLE
Change the footer links from <h4> to <p>

### ISSUE THREAD

**Issue by ewmcelvaney** [CodefortheCarolinas/tax-benefit-website#52]
The footer links being inside of h4 tags is not semantically correct and was flagged by the accessibility checker. Only headings should have any h1 through h6 tag and thus the p tag would be more appropriate and will not throw off screen readers.
